src/StimulusBundle/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.13.0
+
+-   Normalize parameters names given to twig helper 'stimulus_action()'. **BC Break**: previously, parameters given in camelCase (eg. `bigCrocodile`) would be registered by the controller as flatcase (`event.params.bigcrocodile`). Now, they are registered as camelCase (`event.params.bigCrocodile`) as expected.
```

Comment:
Break onto multiple lines when it gets long (around 72ish chars) and indent the other lines the same as the word `Normalize`. Also, slight rewording:

> camelCase (eg. `bigCrocodile`) were incorrectly registered ...

And change the last sentence to:

> This was fixed, which means they are now correctly registered as camelCase (`event.params.bigCrocodile`).

Reply:
And... long live the crocodile  😉

Reply:
Done :smiley: